libs/designsystem/toggle/src/toggle.component.ts

```diff
@@ -58,10 +89,25 @@ export class ToggleComponent implements ControlValueAccessor, OnInit {
     return this.disabled ? 'disabled' : null;
   }
 
+  @HostBinding('class.has-hidden-label') _labelText: string;
+
   @Output() checkedChange = new EventEmitter<boolean>();
 
+  _justify: 'start' | 'end' | 'space-between' = 'start';
+  _labelPlacement: 'end' | 'fixed' | 'stacked' | 'start' = 'start';
+  _hasSlottedContent: boolean;
+
   _pressed = false;
 
+  ngAfterViewInit(): void {
+    this.ionicElementPartHelper.setPart('label', this.ionToggleElement, '.toggle-wrapper');
+    this.ionicElementPartHelper.setPart(
+      'label-text-wrapper',
+      this.ionToggleElement,
+      '.label-text-wrapper'
+    );
+  }
+
```

Comment:
Now that we are at it, consider re-ordering the variables, functions, hooks etc. to match checkbox/radio structure.